source/lib-intro.tex

```diff
@@ -1441,7 +1441,7 @@
 \ref{support.types}      & Types                     & \tcode{<cstddef>}          \\ \rowsep
 \ref{support.limits}     & Implementation properties &
   \tcode{<cfloat>}, \tcode{<climits>}, \tcode{<limits>}, \tcode{<version>} \\ \rowsep
-\ref{cstdint}            & Integer types             & \tcode{<cstdint>}          \\ \rowsep
+\ref{support.arith.types}& Arithmetic types          & \tcode{<cstdint>}, \tcode{<stdfloat>} \\ \rowsep
```

Comment:
Are we sure `<stdfloat>` was freestanding before? I'm not seeing a marker.

Reply:
No, absolutely not, that was just a lack of attention.

I didn't see a good way to keep the actual section heading "Integer types", but maybe we can still use that in this table?

Reply:
The alternative would be to add a third nesting level and have:

```
  Arithmetic types
  +- Integer types
  |  +- General
  |  +- Synopsis
  +- Floating-point types
     +- General
     +- Synopsis

But given that each "general" section would only contain a single fluff paragrah, that seemed excessive.

Reply:
The normative content of the "freestanding" table is the list of headers, not the headings, so I'm fine with just using "integer types" here.

Reply:
Yeah, I think that's a reasonable compromise. Thanks!